metric_learn/lmnn.py

```diff
@@ -25,7 +25,7 @@
 
 
 # commonality between LMNN implementations
-class _base_LMNN(MahalanobisMixin, TransformerMixin):
+class _BaseLMNN(MahalanobisMixin, TransformerMixin):
```

Comment:
Seems like we can also get rid of this base class, right?

Reply:
Ah yes, that's right! I forgot about that, will do

Reply:
done